xdp-filter/tests/test-xdp-filter.sh

```diff
@@ -261,6 +261,36 @@ test_print()
     check_run $XDP_FILTER unload $NS -v
 }
 
+
+run_python_test()
+{
+    if [ "${PYTHON}" = "invalid" ]; then
+        return "$SKIPPED_TEST"
+    fi
+
+    MODULE_PATH=$(realpath --relative-to=. "$TOOL_TESTS_DIR" | sed "s/\//./g")
```

Comment:
Ah, that's where you're using TOOL_TESTS_DIR - please add it in the same commit, then :)